packages/react-ui-components/src/stories/Components/ActionButton.stories.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { EActionButtonType, KvActionButton } from '../../components';
 
 export default {
-	title: "Components/Buttons",
+	title: "Components/Buttons/Action",
```

Comment:
Action should be added?

Reply:
This is more a design choice 😄

The purpose of this change was to have all buttons aggregated under the same `Button` folder as you can see on this [screenshot](https://user-images.githubusercontent.com/17494742/154248291-e6a1c007-e4fb-4db7-b401-151ef39ae879.png).

```
- Buttons
   - ActionButton
   - SwitchButton
```

But I can easily rollback here if you don't think it makes sense 😄

Reply:
I didn't understand until now. 😄  Let's continue with this.